### DISCUSSION TITLE
Hosting Team meeting agenda 2024-11-27

### DISCUSSION THREAD

**Discussion by Crixu** [WordPress/hosting-handbook#279] [Weekly Meetings]
Hosting Team meeting agenda 2024-11-27

---

PUBLIC AGENDA URL: https://make.wordpress.org/hosting/2024/11/27/hosting-team-meeting-agenda-2024-11-27/

PUBLIC SUMMARY URL (if needed): 

# Agenda

WordPress Community
- WordPress Core news
- WordPress Community news

Hosting Team Time
- WordCamp Malaysia 2024 Contributor Day Recap
- Hosting Team Reps for 2025

Hosting Team To-Do

Open Floor
- Anything not in the agenda

Closing
- Next Meeting Times

# Meeting

/here [meeting type="hosting"] Who's here to chat about hosting?

👋 A warm welcome to all WordPress hosting enthusiasts! We're here to dive into the exciting world of WordPress hosting and development. Have questions? Looking for more information? Check out [Make/Hosting](https://make.wordpress.org/hosting/) for comprehensive guides and updates.

🥇 Participants in the [Five for the Future](https://wordpress.org/five-for-the-future/) program, we'd love to recognize your contributions! Please share your WordPress.org username in a thread here 👉

🤗 Newcomers, we're thrilled to have you on board! You are welcome to introduce yourself and share your experience or what you hope to contribute to WordPress. 👉

_Hello, I am Javier Casares, and today I am here leading this meeting. If you have any questions or queries, please do not hesitate to ask. At the end of the agenda, we have an open time to discuss anything else you want [about the team's projects](https://make.wordpress.org/hosting/handbook/get-involved/team-projects/)._

## Agenda

🟢 Here is the agenda for this meeting

[…] Use the agenda above […]

Would anyone like to add to or change any items on the agenda?

OK, let's get started with the WordPress Community news.

### 🟢 WordPress Core news

- [WordPress 6.7.1 Maintenance Release](https://wordpress.org/news/2024/11/wordpress-6-7-1-maintenance-release/)
- [Update on Phase 3: Collaboration efforts](https://make.wordpress.org/core/2024/11/21/update-on-phase-3-collaboration-efforts/)
- [WordPress 6.8 Call for Volunteers](https://make.wordpress.org/core/2024/11/21/wordpress-6-8-call-for-volunteers/)
If you are interested in participating in WordPress 6.8’s[ release squad](https://make.wordpress.org/core/handbook/about/release-cycle/wordpress-release-team-and-focus-leads/) as a lead, please show interest in the comments for the P2, clearly specifying your desired role, by December 6th.

### 🟢 WordPress Community news

🟢 WordPress Community news

[Watch the full recording of the Office hours with Mary and Matt](https://cloudup.com/cXJzq-t-u7o)

[WordPress Contributor Mentorship: 2024 Q4 Cohort Graduates](https://make.wordpress.org/community/2024/11/21/wordpress-contributor-mentorship-2024-q4-cohort-graduates/)

Ok, let's move on to Hosting Team time.

### 🟢 WordCamp Malaysia 2024 Contributor Day Recap

We are happy to note that a total of 7 first time contributors joined the Hosting Table and a few issues have been submitted including a few PRs to the Hosting Team Repository in GitHub.
[WordCamp Malaysia 2024 Contributor Day Recap](https://make.wordpress.org/hosting/2024/11/22/wordcamp-malaysia-2024-contributor-day-recap/)

### 🟢 WordPress Hosting Team Reps

The Call for Nominations for the Wordpress Hosting Team Representatives will close later today. 
[Nominations for Hosting Team Reps 2025](https://make.wordpress.org/hosting/2024/11/13/nominations-for-hosting-team-reps-2024-2/). The call will be closed later today (or tomorrow) at 23:59 UTC. 
You can still nominate community members by:

- Leaving a comment on that post
- Leaving an answer in a thread of this message
- Reach out to the existing team reps Crixu and Javier
- Use the[ anonymous survey](https://crixu4491.survey.fm/hosting-team-rep-survey-2025)

We're aiming to get at least 2 Team Reps with a more spread timezone coverage than currently (Both Javier and Crixu) are in UTC+1 and it would be great to have backups to run meetings, attend Contributor Days and onboard new members.

Once the Call for Nomination is closed, we will start to reach out to the nominated candidates and check their availability for the role. Once confirmed we'll open a Voting Poll through CrowdSignal which will keep open for two weeks. 

## 🟩 Open Floor

Ok, if nothing else, let's move on to the Open Floor. You are welcome to bring any topic up for discussion at this time.

…

### 🟩 Meeting Times

- 2024-12-04 1800 UTC
- 2024-12-11 0900 UTC
- 2024-12-18 1800 UTC

### Goodbye

That wraps up today's meeting. Please keep in mind, our discussions can continue in the [#hosting Slack channel](https://wordpress.slack.com/archives/C3D6T7F8Q) or on [Make/Hosting](https://make.wordpress.org/hosting/) anytime.

I hope you have a wonderful rest of your week! [/meeting]

**Comment by Crixu** [comment]
The Call for Nominations for the Wordpress Hosting Team Representatives will close later today. 
[Nominations for Hosting Team Reps 2025](https://make.wordpress.org/hosting/2024/11/13/nominations-for-hosting-team-reps-2024-2/). The call will be closed later today (or tomorrow) at 23:59 UTC. 
You can still nominate community members by:

- Leaving a comment on that post
- Leaving an answer in a thread of this message
- Reach out to the existing team reps Crixu and Javier
- Use the[ anonymous survey](https://crixu4491.survey.fm/hosting-team-rep-survey-2025)

We're aiming to get at least 2 Team Reps with a more spread timezone coverage than currently (Both Javier and Crixu) are in UTC+1 and it would be great to have backups to run meetings, attend Contributor Days and onboard new members.

Once the Call for Nomination is closed, we will start to reach out to the nominated candidates and check their availability for the role. Once confirmed we'll open a Voting Poll through CrowdSignal which will keep open for two weeks.

**Comment by chaion07** [comment]
I think this should be included in the WordPress Core News

[WordPress 6.8 Call for Volunteers](https://make.wordpress.org/core/2024/11/21/wordpress-6-8-call-for-volunteers/)
If you are interested in participating in WordPress 6.8’s[ release squad](https://make.wordpress.org/core/handbook/about/release-cycle/wordpress-release-team-and-focus-leads/) as a lead, please show interest in the comments for the P2, clearly specifying your desired role, by December 6th.

**Reply by Crixu** [reply]
I would also include [Update on Phase 3: Collaboration efforts](https://make.wordpress.org/core/2024/11/21/update-on-phase-3-collaboration-efforts/)

**Reply by Crixu** [reply]
As well: [WordPress 6.7.1 Maintenance Release](https://wordpress.org/news/2024/11/wordpress-6-7-1-maintenance-release/)

**Comment by Crixu** [comment]
### Community News

**Reply by Crixu** [reply]
[Watch the full recording of the Office hours with Mary and Matt](https://cloudup.com/cXJzq-t-u7o)

**Reply by Crixu** [reply]
[WordPress Contributor Mentorship: 2024 Q4 Cohort Graduates](https://make.wordpress.org/community/2024/11/21/wordpress-contributor-mentorship-2024-q4-cohort-graduates/)

**Reply by chaion07** [reply]
[WordCamp Malaysia 2024 Contributor Day Recap](https://make.wordpress.org/hosting/2024/11/22/wordcamp-malaysia-2024-contributor-day-recap/)

We are happy to note that a total of 7 first time contributors joined the Hosting Table and a few issues have been submitted including a few PRs to the Hosting Team Repository in GitHub.

**Reply by Crixu** [reply]
I moved the CD recap to the hosting team topics :)

**Comment by chaion07** [comment]
Added the public agenda URL to the discussion: https://make.wordpress.org/hosting/2024/11/27/hosting-team-meeting-agenda-2024-11-27/
